html-web-page/cypress.config.mjs

```diff
@@ -1,8 +1,13 @@
 import { defineConfig } from 'cypress';
 
+const port = Number.parseInt(process.env.ROMCAL_APP_PORT ?? '3000', 10);
```

Comment:
Just a question: should we prefer `Number.parseInt(process.env.ROMCAL_APP_PORT, 10) || 3000` to `Number.parseInt(process.env.ROMCAL_APP_PORT ?? '3000', 10)`? In the former one, we are not parsing a string literal, thus (possibly) optimise the code (albeit just a little).

Reply:
Personally because I prefer parsing something non-null or empty